ImageGallery/src/org/sleuthkit/autopsy/imagegallery/datamodel/grouping/DrawableGroup.java

```diff
@@ -113,14 +120,10 @@ public DoubleBinding hashHitDensityProperty() {
      */
     public synchronized long getHashSetHitsCount() {
         if (hashSetHitsCount.get() < 0) {
-            try {
-                hashSetHitsCount.set(fileIDs.stream()
-                        .map(ImageGalleryModule.getController().getHashSetManager()::isInAnyHashSet)
```

Comment:
ImageGalleryModule now unused import.

also copyright update

Reply:
I removed the unused import and updated the copyright.